apps/docs/src/common/pages/accessibility/resources.tsx

```diff
@@ -30,8 +30,8 @@ const Page: FC<PageProps> = ({ location }) => (
       Guidance
     </h2>
     <ul className="plain">
-      <li><A href="https://collaboration.homeoffice.gov.uk/display/A11y/Accessibility+Standard">Home Office Accessibility Standard v1.0 Nov 2020</A></li>
-      <li><A href="https://homeofficegovuk.sharepoint.com/:w:/s/designops/EeB8PFKt6CNEmzfijRoZAO8BudZ1A8ths4rpx4h8qykZDA?e=Mwm90N">Accessibility checklist for designers</A></li>
+      <li><A href="https://collaboration.homeoffice.gov.uk/display/A11y/Accessibility+Standard">Home Office Accessibility Standard v1.0 Nov 2020</A> (Home Office Confluence link)</li>
+      <li><A href="https://homeofficegovuk.sharepoint.com/:w:/s/designops/EeB8PFKt6CNEmzfijRoZAO8BudZ1A8ths4rpx4h8qykZDA?e=Mwm90N">Accessibility checklist for designers</A> (Home Office Sharepoint link)</li>
```

Comment:
I wonder if it's best to call these "private links" or something.

Reply:
This could be an option, although wonder if that might create some confusion (inc for new starters) around who the links are private for?

Reply:
I noticed on another PR that @karypun asked for this wording.

I was thinking it _might_ be confusing to external users. (But then those aren't our main audience.)

As long as you are both happy with it.